packages/server/lib/controllers/spec.ts

```diff
@@ -37,15 +40,16 @@ module.exports = {
 
     return preprocessor
     .getFile(spec, config)
-    .then((filePath) => {
+    .then((filePath: string) => {
       debug('sending spec %o', { filePath })
       const sendFile = Promise.promisify(res.sendFile.bind(res))
 
+      // @ts-expect-error TODO: sendFile doesn't accept any args, yet here one is passed
       return sendFile(filePath)
```

Comment:
Sanity check here….what is this supposed to do? Clearly this method doesn’t accept an argument, but we passed an arg, but why are we passing down filePath if this is never used. Is this method supposed to accept this arg??

Reply:
According to the documentation, [`sendFile`](https://github.com/expressjs/express/blob/master/lib/response.js#L376) accepts a path. I also stepped into the function and it accepts the path.